esrally/rally.py

```diff
@@ -661,6 +661,7 @@ def add_track_source(subparser):
     )
 
     for p in [
+        parser,
```

Comment:
Is this actually needed given the `sys.exit(0)` call?

Reply:
Not anymore, you're right. I think I put this in place before I added the early exit. Removed.